src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineWindowsLauncher.java

```diff
@@ -46,44 +44,14 @@
   // TODO: make this configurable
   public static final Integer SSH_PORT = 22;
   public static final Integer SSH_TIMEOUT = 10000;
-  private static final Logger LOGGER = Logger.getLogger(ComputeEngineLinuxLauncher.class.getName());
   private static int bootstrapAuthTries = 30;
   private static int bootstrapAuthSleepMs = 15000;
 
   public ComputeEngineWindowsLauncher(
       String cloudName, Operation insertOperation, boolean useInternalAddress) {
     super(cloudName, insertOperation.getName(), insertOperation.getZone());
     this.useInternalAddress = useInternalAddress;
-  }
-
-  protected void log(
-      Level level, ComputeEngineComputer computer, TaskListener listener, String message) {
-    try {
-      ComputeEngineCloud cloud = computer.getCloud();
-      cloud.log(LOGGER, level, listener, message);
-    } catch (CloudNotFoundException cnfe) {
-      ComputeEngineComputerLauncher.log(
-          LOGGER, Level.SEVERE, listener, "FATAL: Could not get cloud");
-    }
-  }
-
-  protected void logException(
-      ComputeEngineComputer computer, TaskListener listener, String message, Throwable exception) {
-    try {
-      ComputeEngineCloud cloud = computer.getCloud();
-      cloud.log(LOGGER, Level.WARNING, listener, message, exception);
-    } catch (CloudNotFoundException cnfe) {
-      ComputeEngineComputerLauncher.log(
-          LOGGER, Level.SEVERE, listener, "FATAL: Could not get cloud");
-    }
-  }
-
-  protected void logInfo(ComputeEngineComputer computer, TaskListener listener, String message) {
-    log(Level.INFO, computer, listener, message);
-  }
-
-  protected void logWarning(ComputeEngineComputer computer, TaskListener listener, String message) {
-    log(Level.WARNING, computer, listener, message);
+    this.pathSeparator = "\\";
```

Comment:
This introduces tight coupling between the parent and child classes. Suggest having an abstract method: getPathSeparator instead.

Reply:
Done